models/calendar.yml

```diff
@@ -57,7 +57,17 @@
 
     - 8:30 PM: Performer TBA
 
-    - 8:45 PM: Speaker TBA
+    - 8:45 PM:
+        type: Talk
+        title: If You Only Make Chrome Extensions, You're Killing The Open Web
+        emoji: ❄
```

Comment:
![image](https://cloud.githubusercontent.com/assets/748504/14032058/a0121d8e-f1cd-11e5-9ebb-37618ee67f14.png)
Unfortunately, the snowflake emoji isn't rendered correctly by Chrome. Would you mind picking another emoji?